### DISCUSSION TITLE
How to add custom repository URL for apt?

### DISCUSSION THREAD

**Discussion by Sunderland93** [go-debos/debos#601] [Q&A] [answered]
Hello. Сould you advise me how can add custom Debian repo in recipe? For example, I want to use my repo on OBS (Open Build Service), which is looks like http://download.opensuse.org/repositories/home:/Sunderland93:/repo

**Comment by stephenstech-ca** [comment]
Is there any update regarding this?  This would be very useful.

**Comment by obbardc** [comment, marked as answer]
You can add custom apt repository by modifying files under `/etc/apt` e.g.:

```
  - action: run
    description: Add custom apt sources
    command: echo "deb http://site.example.com/debian distribution component1 component2 component3" >> ${ROOTDIR}/etc/apt/sources.list
```

You can also use `overlay` actions to add new files into e.g. `/etc/apt/sources.list.d` as appropriate. Hope that helps.

**Comment by stephenstech-ca** [comment]
Thanks Christopher!  That's very helpful.
